src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.InlineRenameLocationSet.cs

```diff
@@ -7,22 +7,21 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Rename;
-using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Rename.ConflictEngine;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
     internal abstract partial class AbstractEditorInlineRenameService
     {
         private class InlineRenameLocationSet : IInlineRenameLocationSet
         {
-            private readonly RenameLocations _renameLocationSet;
+            private readonly LightweightRenameLocations _renameLocationSet;
```

Comment:
this part of inlinerename is on the host side, so we only want it to hold onto lightweight stuff.